projectsmember/Profile.aspx.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace projectsmember
+{
+    public partial class Profile : System.Web.UI.Page
+    {
+        protected void Page_Load(object sender, EventArgs e)
+        {
+
+
+            if (Session["status"] != null &&
+                Session["status"].ToString() == "Login_User")
```

Comment:
as mentioned in earlier comment, if I set the "status" variable in my browser and the value to the pre-defined "Login_User" , I can be the admin of this website! note that the code is also public, so anyone can know about these pre-defined values.

Reply:
Done !